core/src/test/java/org/testcontainers/junit/ParameterizedDockerfileContainerTest.java

```diff
@@ -26,39 +17,33 @@
 public class ParameterizedDockerfileContainerTest {
 
     @Parameterized.Parameters(name = "{0}")
-    public static Object[] data() {
-        return new Object[] { "alpine:3.2", "alpine:3.3" };
+    public static Object[][] data() {
+        return new Object[][] {
+                { "alpine:3.2", "3.2"},
+                { "alpine:3.3", "3.3"},
+                { "alpine:3.4", "3.4"}
```

Comment:
probably should add `3.5` and `3.6` as well

Reply:
Sure, why not 😄

Reply:
Done